version.php

```diff
@@ -26,7 +26,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version = 2020050504;
+$plugin->version = 2020050506;
```

Comment:
Avoid a 2 bump, it might be better to leave this at 505, and whichever PR gets merged first will win, and you can just bump the other one before we merge